JS/Lesson35/onur/my-app/src/components/NewRecipe/index.jsx

```diff
@@ -57,154 +62,204 @@ export const NewRecipe = () => {
 					processedData[`strMeasure${i}`] = "";
 				}
 			}
-
+			return null;
 		});
 		delete processedData.measures
 
-
 		processedData.dateModified = new Date().toString();
 
-		reset();
+		const allId = recipeContext.map(recipe => recipe.idMeal)
+		const newId = Math.max(...allId) + 1
```

Comment:
Using timestamp would generate you a unique enough id 
```suggestion
		const newId = Date.now();
```